modules/ROOT/pages/autocompleter.adoc

```diff
@@ -23,7 +23,12 @@ The two arguments this method take are:
 [cols="1,2,1,4",options="header"]
 |===
 |Name |Value |Requirement |Description
-|ch |string (of one character) |Required |The character to trigger the autocompleter.
+|ch |`+string+` |Required |The character to trigger the autocompleter.
```

Comment:
You haven't fixed this fully... it's still missing the part about trigger replaced this and should be used instead.

- [ ] Correct details

Reply:
I see actually you have this in `trigger` actually... it'd make more sense to have it here, as this is the option we don't want people to use moving forward.

Reply:
It is here, this table cell follows three lines as the partial cannot be on a line with anything else

Reply:
It's in the wrong spot as I said in my last message.

Reply:
Ahh right, you did have something for both which I missed sorry. That said you really don't need both and we don't normally have it for both so you can remove the `trigger` message as I left a suggest for.

Reply:
If this current change doesn't fix what you are asking then I'm sorry I just don't understand. The first line of this option (and of the `trigger` option) describes its use, then the line after it has the deprecation (or requires) partial since that cannot go on the same line as anything else, and then the third line states the correct option to use. I put the final part after the partial since it makes more sense that way, as it is stating the replacement immediately after the deprecation (or requires) notice.